cardinal/ecs/gamestate/ecb_test.go

```diff
@@ -19,19 +21,21 @@ import (
 	"github.com/redis/go-redis/v9"
 
 	"pkg.world.dev/world-engine/cardinal/ecs"
-	"pkg.world.dev/world-engine/cardinal/ecs/ecb"
 	"pkg.world.dev/world-engine/cardinal/types/component"
 	"pkg.world.dev/world-engine/cardinal/types/entity"
 )
 
-func newCmdBufferForTest(t *testing.T) *ecb.Manager {
+func newCmdBufferForTest(t *testing.T) *gamestate.EntityComponentBuffer {
```

Comment:
There's a mix of CommandBuffer and ComponentBuffer in this PR. It should all be CommandBuffer or CmdBuffer.

Reply:
@jerargus fixed